agent/handlers/task_server_setup_test.go

```diff
@@ -1189,6 +1189,7 @@ func TestV4TaskMetadata(t *testing.T) {
 		state.EXPECT().TaskByArn(taskARN).Return(task, true).AnyTimes(),
 		state.EXPECT().ContainerMapByArn(taskARN).Return(containerNameToDockerContainer, true),
 		state.EXPECT().TaskByArn(taskARN).Return(task, true).AnyTimes(),
+		state.EXPECT().PulledContainerMapByArn(taskARN).Return(nil, true),
```

Comment:
could we return a pulled container in the mock `PulledContainerMapByArn` and verify in the response?

Reply:
`TestV4TaskMetadataWithPulledContainers` is added to agent/handlers/task_server_setup_test.go. Thanks!